media/js/src/GraphMapping.js

```diff
@@ -259,19 +356,19 @@ const defaultEvaluation = {
     intersection3_horiz_line_label: '',
     intersection3_vert_line_label: '',
 
-    line_1_slope: 1,
-    line_2_slope: -1,
-    line_3_slope: 999,
-    line_4_slope: -1,
-
-    line_1_offset_x: 0,
-    line_1_offset_y: 0,
-    line_2_offset_x: 0,
-    line_2_offset_y: 0,
-    line_3_offset_x: 0,
-    line_3_offset_y: 0,
-    line_4_offset_x: 0,
-    line_4_offset_y: 0,
+    line_1_slope: '1.0000',
```

Comment:
If we are hard-coding numbers as default here, they should be of the `number` type rather than `string`.